EdFi.Buzz.Etl/eng/build.js

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+// This build script runs a number of yarn commands in an OS-agnostic fashion
+// with the help of Node.js.
+
+const { execSync } = require('child_process');
+
+const steps = [
+  'yarn synp -s yarn.lock -f',
+  'yarn copyfiles .env.example package-lock.json package.json config/**/* processors/**/*  src/**/* sql/**/* tasks/**/* dist',
```

Comment:
This is a larger set of files than I would have expected. The original use of this was to copy a few files that `yarn build` did not load into `dist`, but which are required on the deployment server. I'm particularly curious about `src/**/*` - why would source TypeScript files need to be put into the distribution?

Reply:
I cannot speak for Xavier, but resources for completing the database ETL in TypeScript were having an impact on the delivery time. I opted for straight JavaScript to ensure I could complete it on time.

Reply:
OK